arangod/Cluster/TraverserEngine.cpp

```diff
@@ -46,6 +46,11 @@
 
 #include <string_view>
 
+// TODO: the lookupToken should probably be somewhere else?
```

Comment:
Wanna address this in this PR.

Reply:
Not really because I do not know yet where it should go; maybe into a helper class?